### DISCUSSION TITLE
Git clone issues

### DISCUSSION THREAD

**Discussion by ramlohith** [OpenDDS/OpenDDS#2681] [General]
Hello,

I get the following error when I clone this repository: 

remote: Enumerating objects: 213268, done.
remote: Counting objects: 100% (4654/4654), done.
remote: Compressing objects: 100% (1750/1750), done.
fatal: protocol error: bad line length character: ngin KiB/s   
error: inflate: data stream error (incorrect header check)s   
fatal: pack has bad object at offset 80871: inflate returned -3
fatal: index-pack failed

I'm running this clone for a yocto build (using the bitbake engine) and I cannot do a shallow clone of the repository. Also, I tried just running git clone on my machine and I face the same issue. Any known fix to this? Has anyone else encountered this issue? 

Thanks!

**Comment by iguessthislldo** [comment]
What is the command you're using to do the clone?

**Comment by jwillemsen** [comment]
There is a [stackoverflow question](https://stackoverflow.com/questions/21277806/fatal-early-eof-fatal-index-pack-failed) with several possible causes.
